src/index.js

```diff
@@ -1,104 +1,243 @@
-          "hp": 10,
-          "gold": 100,
-          "name": name,
-          "factions": [
-            {
-              "factions_id": {
-                "id": facId,
-                "name": faction
-              }
-            }
-          ],
-          "equipments": []
-        };
-        try{
-          const print = await postApi(user, "characters?fields=*,equipments.equipments_id.*,factions.factions_id.*")
-          console.log(print);
-        } catch (error){
-          console.log(error);
-        }
+  const quests = async () => {
+    clear();
+    console.log("select a quest:\n");
+    let questList = await getApi("tasks");
+    let bugList = await getApi("bugs");
+    questList.forEach((questn, index) => {
+      let bugHp = [1];
+      questn.bugs.forEach((monstern) => {
+        let bugn = bugList.filter((bug) => bug.id == monstern);
+        bugHp.push(bugn[0].hp);
+      });
+      bugHp.shift();
+      console.log(
+        `${index + 1}: ${questn.name}\nDescription: ${
+          questn.description
+        }\nBug hp: ${bugHp}\nReward: ${questn.reward}\n`
+      );
+    });
+  };
+
+  const store = async (costumer) => {
+    clear();
+    console.log(
+      `You want it? It's your's my friend! As long as you have enough gold.\nGold: ${costumer.gold}\n`
+    );
+    let equipList = await getApi("equipments");
+    let buy = 1;
+    while (buy != 0) {
+      equipList.forEach((equipn, index) => {
+        console.log(
+          `${index + 1} - ${equipn.name}\nPrice: ${equipn.value} gold\nStat: +${
+            equipn.affected_amount
+          } ${equipn.affected_attribute}\n`
+        );
+      });
+      buy = await read("0 - Exit");
+      if (buy == 0) {
+        clear();
+        console.log("Take care out there my friend!");
+        return 0;
+      }
+      let pick = equipList[buy - 1];
+      if (pick.value <= costumer.gold) {
+        costumer.equipments.push(pick);
+        costumer.gold -= pick.value;
+        clear();
+        console.log(
+          `Thanks for your Purchase! Do you want anything else?\nGold: ${costumer.gold}\n`
+        );
+      } else {
+        console.log(
+          `Sorry, ${costumer.name}. I can't give credit. Come back when you are a little... mmmmmm richer!`
+        );
+      }
+    }
+  };
+
+  const stats = async (you) => {
+    clear();
+    let fatk = you.atk,
+      fdef = you.def,
+      fagi = you.agi,
+      attribute,
+      amount;
+    you.equipments.forEach((item) => {
+      amount = item.equipments_id.affected_amount;
+      attribute = item.equipments_id.affected_attribute;
+      if (attribute == "atk") {
+        fatk += amount;
+      } else if (attribute == "def") {
+        fdef += amount;
+      } else {
+        fagi += amount;
+      }
+    });
+    console.log(`Attack: ${fatk}\nDefense: ${fdef}\nAgility: ${fagi}\n`);
+  };
+
+  const menu = async () => {
+    clear();
+    let name, nav;
+    name = await read("Do i remember you? Tell me your name.\n");
```

Comment:
couldn't `name` be a `const` in this situation?

Reply:
No, because you could log off and log in in the same execution.

Reply:
oh ok, ignore then